src/index.html

```diff
@@ -22,6 +22,362 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header_logo"
+        href="#logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#Apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="nav__item">
```